docker-compose.yml

```diff
@@ -332,4 +332,28 @@ services:
     ulimits:
       nofile:
         soft: 262144
-        hard: 262144
\ No newline at end of file
+        hard: 262144
+
+# Web-client configuration
+  web-client:
+    image: your-docker-registry/hyperswitch-web:latest # Need to enter hyperswitch-web-image 
```

Comment:
We don't have the pre-built images available on Docker Hub today. Could you update your PR to handle it by building it instead?